src/styles/blocks/card.scss

```diff
@@ -0,0 +1,35 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use multiples of 4 for margin and padding values. It would be better to adjust the padding value to follow this rule.